VideoApp/VideoApp/Stores/Auth/InternalAuthStore.swift

```diff
@@ -78,19 +88,33 @@ extension FirebaseAuthStore: GIDSignInDelegate {
         )
```

Comment:
I think this is interesting. There is both an access token and ID token. I'm not sure if using ID token elsewhere with Firebase is optimal. Access token is generally more common I think. But I don't know enough about Firebase auth yet and it's clearly working for us. Just interesting.